app/models/spree/address.rb

```diff
@@ -12,7 +12,6 @@ class Address < ApplicationRecord
 
     validates :firstname, :lastname, :address1, :city, :country, presence: true
     validates :zipcode, presence: true, if: :require_zipcode?
-    validates :phone, presence: true, if: :require_phone?
```

Comment:
What to you think @andrewpbrett (or @Matt-Yorkley ) ? ;)

Reply:
This will apply to all addresses in all contexts, yes. I think we need some clarification from the product team to find out whether or not that's what we want to do.

Reply:
@jaycmb Regarding this modification, and specially issue #8008 we're wondering if making phone number field optional applies only to split checkout or in all contexts when address is required. Thanks. 🙏

Reply:
Where else is the shopper asked for their phone number but in the checkout?
We only talking about shopper facing modifications, right?

Reply:
Okay, clarified in delivery train: we are talking **not** only about a shopper facing modification but really for all contexts, i.e. also for enterprise addresses.
As discussed, we go with this change for now as there seems to be no use case where phone number needs to be mandatory on the enterprise side.
If such a case should turn up, the change will be addressed in a new issue.